client/data/settings/actions.js

```diff
@@ -0,0 +1,66 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { dispatch, select } from '@wordpress/data';
+import { apiFetch } from '@wordpress/data-controls';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
```

Comment:
As I said in the `reducer.js` file... what do you think about renaming this to something more verbose like `ACTION_TYPES`?

Reply:
:heart: